anoma/namada-interface
# feat/682: adding crypto functions to sdk package

Resolves #682 

- [x] Crypto functionality from `packages/crypto` added to `packages/sdk`
- [x] Connected vault service in extension to this package
  - [x] tests should cover example `CryptoRecord` instances as stored by the extension

### Testing

- Checkout `main`, build and install dev extension (`yarn start:chrome`), and set up some accounts
- Checkout this branch: `feat/682-crypto-sdk` and start (`yarn start:chrome`), refresh the extension in `My Extension`
- Ensure that all functions related to managing existing keys works (where encryption and decryption is concerned). And of course, installing a new extension from this branch should function as expected. The goal is to ensure that an extension store created with the previous `crypto` functionality works perfectly with the changes in this PR!
